From ef3b70be9de8e1c798db47b627b465b0a257e693 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Thu, 21 Sep 2023 20:56:50 +0200
Subject: [PATCH] Trivial fix for GLib 2.78

generated/glib/StringG.d(137): Error: constructor `glib.StringG.StringG.this(string init)` conflicts with previous declaration at generated/glib/StringG.d(78)
---
 src/APILookupGLib.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/APILookupGLib.txt b/src/APILookupGLib.txt
index d037f7a..f66969d 100644
--- a/src/APILookupGLib.txt
+++ b/src/APILookupGLib.txt
@@ -1313,6 +1313,7 @@ code: end
 
 struct: String
 class: StringG
+version 2.78: noCode: new_take
 
 struct: Thread
 noCode: new
